SPEC.md

```diff
@@ -504,30 +504,30 @@ JSON Schema for the Image Manifest (app image manifest, ACI manifest), conformin
 }
 ```
 
-* **acKind** (required) must be set to "ImageManifest"
-* **acVersion** (required) represents the version of the schema specification that the manifest implements (string, must be in [semver](http://semver.org/) format)
-* **name** (required) is used as a human readable index to the container image. (string, restricted to the AC Name formatting)
-* **labels** (optional) list of label objects (where the *name* is restricted to the AC Name formatting and *value* is an arbitrary string). Label names must be unique within the list, and (to avoid confusion with the image's name) cannot be "name". Labels are used during image discovery and dependency resolution. Several well-known labels are defined:
+* **acKind** (string, required) must be set to "ImageManifest"
+* **acVersion** (string, required) represents the version of the schema specification that the manifest implements (string, must be in [semver](http://semver.org/) format)
+* **name** (string, required) is used as a human readable index to the container image. (string, restricted to the AC Name formatting)
+* **labels** (list of objects, optional) are used during image discovery and dependency resolution. The listed objects should have two key-value pairs: *name* is restricted to the AC Name formatting and *value* is an arbitrary string. Label names must be unique within the list, and (to avoid confusion with the image's name) cannot be "name". Several well-known labels are defined:
```

Comment:
"two key-value pairs"
The listed objects should have two key-value pairs -> List objects are key-value pairs